### DISCUSSION TITLE
Is v1.0.0 a release version?

### DISCUSSION THREAD

**Discussion by dtzWill** [Pezmc/BibLatex-Check#57] [Q&A] [answered]
Hi!

The tag v1.0.0 isn't listed as a release, is this a "real" release?
I ask because I'm updating a package I maintain for this, and wasn't sure.

Is v1.0.0 okay to ship, or should folks stick with v0.4.0?
(I'm including the two fixes since the release as well, FWIW)

Thanks for this, btw! :+1:

**Comment by Pezmc** [comment, marked as answer]
Releases were out of date, I've just cut [v1.0.1](https://github.com/Pezmc/BibLatex-Check/releases/tag/v1.0.1), and that's fine to use (it's currently inline with main).
